website/content/docs/troubleshoot/troubleshoot-services.mdx

```diff
@@ -0,0 +1,136 @@
+---
+layout: docs
+page_title: Service-to-service troubleshooting overview
+description: >-
+  Consul includes a built-in tool for troubleshooting communication between services in a service mesh. Learn how to use the `consul troubleshoot` command to validate communication between upstream and downstream Envoy proxies on VM and Kubernetes deployments.
+---
+
+# Service-to-service troubleshooting overview
```

Comment:
can you also update the CLI reference pages to link to this page?

this is the merged PR with the CLI reference docs: https://github.com/hashicorp/consul/pull/16284